core/static/schedule/style.css

```diff
@@ -99,6 +99,12 @@ ol {
   list-style-type: none;
 }
 
+.about {
+  font-family: "Open Sans", "Helvetica Neue", "Helvetica", "Roboto", "Arial", sans-serif;
+  font-size: 18px;
+
```

Comment:
It's a mistake. I was trying to do something and forgot to erase this line. My bad.